app/screens/AppNavigator/screens/Balances/screens/SendScreen.tsx

```diff
@@ -112,7 +113,13 @@ export function SendScreen ({ route, navigation }: Props): JSX.Element {
       />
       <AmountRow
         fee={fee}
-        token={token} control={control} onMaxPress={async (amount) => {
+        token={token}
+        control={control}
+        onMaxPress={async (amount) => {
```

Comment:
aren't they doing the same?

Reply:
added it before i refactor the button, removed now 😅